content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+button. If not already in the kanban view for the job position to add an applicant to, navigate to
+the main recruitment dashboard, and click the :guilabel:`New Applicants` smart button on the job
+position card that the applicant should be added to. Then, click on the small :guilabel:`+ (Quick
+add)` icon in the top right of the :guilabel:`Initial Qualification` stage to quickly add a new
+applicant.
+
+Enter the following information on the card:
+
+- :guilabel:`Subject/Application Name`: This is the only required field. Enter the title for the
+  card. Typically this is the applicant's name and job position being applied to, for example:
+  `Laura Smith - Experienced Developer`. This field is not visible in the kanban view, but is
+  visible on the individual card.
+- :guilabel:`Applicant's Name`: Enter the applicant's name.
+- :guilabel:`Email`: Enter the applicant's email address.
+- :guilabel:`Applied Job`: The current job position populates this field. If needed, the job
+  position can be changed by selecting a different position from the drop-down menu. If a different
+  job position is selected, after the card is created, the card will appear in the selected job
+  position.
+
+After the information is entered, click :guilabel:`Add`. The applicant appears in the list, and a
+new blank applicant card appears. Click either the :guilabel:`🗑️ (delete)` icon or click anywhere
+on the screen to close the card.
+
+.. image:: recruitment/quick-add.png
+   :align: center
+   :alt: All the fields for a new applicant form entered when using the Quick Add option.
+
+Create
+******
+
+Add a new applicant including all the relevant information using the :guilabel:`Create` button. If
+not already in the kanban view for the job position to add an applicant to, navigate to the main
+recruitment dashboard, and click the :guilabel:`New Applicants` smart button on the job position
+card that the applicant should be added to. Then, click the :guilabel:`Create` smart button in the
+top left of the kanban view and an applicant form loads.
+
+Certain fields on the applicant card may be pre-populated, depending on how the job position is
+configured. Typically, the :guilabel:`Job` section, as well as the :guilabel:`Recruiter` field, are
+all pre-populated, but this may vary.
+
+Enter the following information on the new applicant form. Note that not all fields listed below may
+be visible. Depending on installed applications and configurations, some fields may not be
+displayed.
+
+Applicant section
+^^^^^^^^^^^^^^^^^
+
+- :guilabel:`Subject/Application Name`: This is the only required field. Enter the title for the
+  card. Typically this is the applicant's name and job position being applied to, for example:
+  `Laura Smith - Experienced Developer`. This field is not visible in the kanban view, but is
+  visible on the individual card.
+- :guilabel:`Applicant's Name`: Enter the applicant's name.
+- :guilabel:`Email`: Enter the applicant's email address.
```

Comment:
CC Email also appears in dev mode
![2023-10-23_14-06](https://github.com/odoo/documentation/assets/91896312/4f43f5be-dfe5-47fd-9289-3a2cc5c1f3de)
![2023-10-23_11-59](https://github.com/odoo/documentation/assets/91896312/5e6367d6-ab05-4851-a841-90b3a048eaa9)

Reply:
Noted and added that- I typically only put in directions for the default, I am never in dev mode, so I didn't think to mention or look for that. Do we typically add all the exclusions for Dev mode in our docs? I have not done so yet, unless the user must be in dev mode to do something - otherwise, I don't mention it.